README.md

```diff
@@ -165,13 +175,13 @@ If the value for `includePath` had been set to true then the output structure wo
 
 `srcFolder` is the only option that must be supplied. This specifies the folder or folders that are to be processed by the build tools.
```

Comment:
Should `srcFolder` here be `srcFolders`? Or is `srcFolders` wrong where it is found?

Reply:
fixed